external/boost/boost/fusion/container/set/detail/value_of_data_impl.hpp

```diff
@@ -0,0 +1,24 @@
+/*=============================================================================
+    Copyright (c) 2009 Christopher Schmidt
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+==============================================================================*/
+
+#ifndef BOOST_FUSION_CONTAINER_SET_DETAIL_VALUE_OF_DATA_IMPL_HPP
+#define BOOST_FUSION_CONTAINER_SET_DETAIL_VALUE_OF_DATA_IMPL_HPP
+
+#include <boost/fusion/support/config.hpp>
+
+namespace boost { namespace fusion { namespace extension
```

Comment:
file is located in `external/boost/boost/....` is it correct (the double )?

Reply:
external/boost/
you only need to build boost.filesystem and boost.system. All boost headers are in include/boost